### DISCUSSION TITLE
Comparision with original whisper timestamps?

### DISCUSSION THREAD

**Discussion by naymaraq** [linto-ai/whisper-timestamped#159] [Q&A]
Hi,

I would like to test timestamps algorithm by calculating Detection Error Rate. I saw in the original Whisper-repo, that  there is an command line option `--word-timestamps`.  I am wondering is anyone compare these two implementations (whisper-timestamped and original whisper). And how they differ?

**Comment by Jeronymous** [comment]
We did some comparison which show that timestamps of whisper-timestamped tend to be a bit more accurate than the ones in original whisper.

Those results were not fully published, but there are some details in : ["Transcribing And Aligning Conversational Speech: A Hybrid Pipeline Applied To French Conversations" (ASRU 2023)](https://github.com/linto-ai/whisper-timestamped/files/13898238/2023261832.pdf)
Precision of openai-whisper in terms of timestamps is similar to the one of faster_whisper (that re-implements the same algorithm).

A direct comparison of openai-whisper and whisper-timestamped is difficult, because whisper-timestamps offers additional options such as the VAD (which is critical to obtain good results).
